compiler/rustc_trait_selection/src/solve/trait_goals.rs

```diff
@@ -513,6 +513,20 @@ impl<'tcx> assembly::GoalKind<'tcx> for TraitPredicate<'tcx> {
         // `DiscriminantKind` is automatically implemented for every type.
         ecx.evaluate_added_goals_and_make_canonical_response(Certainty::Yes)
     }
+
+    fn consider_builtin_destruct_candidate(
+        ecx: &mut EvalCtxt<'_, 'tcx>,
+        goal: Goal<'tcx, Self>,
+    ) -> QueryResult<'tcx> {
+        if !goal.param_env.is_const() {
+            // `Destruct` is automatically implemented for every type in
+            // non-const environments.
+            ecx.evaluate_added_goals_and_make_canonical_response(Certainty::Yes)
+        } else {
+            // FIXME: Implement this when we get const working in the new solver
```

Comment:
```suggestion
            // FIXME(trait-system-refactor): Implement this when we get const working in the new solver
```

Reply:
I think that's implied in all FIXMEs in this subdirectory of `rustc_trait_selection`.

Also, conventionally we've been adding `FIXME(-Ztrait-solver=next)`

Reply:
Sounds good to me :)